src/anyio/from_thread.py

```diff
@@ -114,10 +135,8 @@ def started(self, value: object = None) -> None:
 class BlockingPortal:
     """An object that lets external threads run code in an asynchronous event loop."""
 
-    def __new__(cls) -> 'BlockingPortal':
-        return get_asynclib().BlockingPortal()
-
     def __init__(self) -> None:
+        self.token = current_token()
```

Comment:
@agronholm I think this should be public so I can run


```python
anyio.from_thread.run_sync(whatever, asynclib_token=portal.token)
```

Reply:
I can always get it from:

`asynclib_token = portal.call(anyio.current_token)`